datawolf-executor-commandline/src/main/java/edu/illinois/ncsa/datawolf/executor/commandline/CommandLineExecutor.java

```diff
@@ -173,6 +173,12 @@ public void execute(File cwd) throws AbortException, FailedException {
 
                             // Create a folder for the datasets
                             File inputFolder = new File(filename);
+                            if (inputFolder.exists()) {
```

Comment:
That's a good point. Since we know the tools run in the temp folder, I can add an additional check in the conditional that the inputFolder starts with what is returned by System.getProperty("java.io.tmpdir"). If the temp folder is /tmp then the inputFolder should start with /tmp